emails/views.py

```diff
@@ -314,19 +315,24 @@ def _sns_message(message_json):
             'email_to': to_address,
             'display_email': display_email,
             'SITE_ORIGIN': settings.SITE_ORIGIN,
+            'has_attachment': bool(attachments),
+            'faq_page': settings.SITE_ORIGIN + reverse('faq')
         })
         message_body['Html'] = {'Charset': 'UTF-8', 'Data': wrapped_html}
 
     if text_content:
         incr_if_enabled('email_with_text_content', 1)
-        attachment_not_supported = ''
+        attachment_msg = (
+            'Emails with a maximum size (including headers) of 150KB '
+            'have attachments support. To learn more visit {site}{faq}\n'
+        ).format(site=settings.SITE_ORIGIN, faq=reverse('faq'))
         relay_header_text = (
             'This email was sent to your alias '
             '{relay_address}. To stop receiving emails sent to this alias, '
             'update the forwarding settings in your dashboard.\n'
             '{extra_msg}---Begin Email---\n'
         ).format(
-            relay_address=display_email, extra_msg=attachment_not_supported
+            relay_address=display_email, extra_msg=attachment_msg
         )
         wrapped_text = relay_header_text + text_content
```

Comment:
It just occurred to me that we could use django templates for the plain text email too. I.e., we could have `emails/templates/emails/wrapped_email.txt`, render it with Django, and send it in `message_body['Text'] ?

Reply:
Forgive me, I'm not entirely sure what I am checking here. One query I have given the tooltip text, when we say " Attachments and emails larger than 150KB are not currently supported and will not be forwarded." Do we mean either:
- Attachments are not supported
or
- Attachments larger than 150KB are not supported

Reply:
@mcroud no worries and great questions. What we wanted to say was that any emails larger than 150KB will not be forwarded to the user due to the limit.